api/src/main/java/io/strimzi/api/kafka/model/template/PodTemplate.java

```diff
@@ -182,6 +184,16 @@ public void setEnableServiceLinks(Boolean enableServiceLinks) {
         this.enableServiceLinks = enableServiceLinks;
     }
 
+    @Pattern("^([0-9.]+)([eEinumkKMGTP]*[-+]?[0-9]*)$")
```

Comment:
Assuming you copy+paste from some other `@Pattern` use site which uses the same regex: Can we make this String a compile-time constant somewhere?